### PR TITLE
fix lint

### PR DESCRIPTION
**Pull request by emnlpstr**
Removed all lint errors

## Checklist

- [x] I followed the indications in the [CONTRIBUTING](CONTRIBUTING.md)
- [x] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?
- [x] Ready for review! :rocket:

## Fixes

- Fixes #

### CODE REVIEW

**Review by lollomag — APPROVED** [review]

File: src/components/hero/HeroImage.jsx [hunk 1]
```diff
@@ -134,3 +140,19 @@ export const HeroImage = ({
...
+  firstInternal: PropTypes.any,
+  firstButtonHref: PropTypes.any,
+  firstButtonLabel: PropTypes.any,
+  firstButtonAriaLabel: PropTypes.any,
+  secondButtonHref: PropTypes.any,
+  secondButtonLabel: PropTypes.any,
+  secondButtonAriaLabel: PropTypes.any,
+  imageUrl: PropTypes.any,
+  imageAlt: PropTypes.any,
+  heroTitleId: PropTypes.any,
```
Comment (sebbalex) [inline src/components/hero/HeroImage.jsx:157]: same here

File: src/components/AccordionButtonFull.jsx [hunk 2]
```diff
@@ -342,3 +343,20 @@ export const AccordionButtonFull = (props) => {
...
+  money: PropTypes.any,
+  tags: PropTypes.any,
+  description: PropTypes.any,
+  stalls: PropTypes.any,
+  accessLabel: PropTypes.any,
+  accessSectionId: PropTypes.any,
+  moreInfoLabel: PropTypes.any,
+  moreInfoLink: PropTypes.any,
+  data: PropTypes.any,
+  active: PropTypes.any,
```
Comment (sebbalex) [inline src/components/AccordionButtonFull.jsx:360]: same here

File: src/components/DesktopSwiper.jsx [hunk 3]
```diff
@@ -222,5 +222,7 @@ DesktopSwiper.propTypes = {
   desktopNavigation: PropTypes.bool,
   navigation: PropTypes.bool,
   className: PropTypes.any,
+  title: PropTypes.any,
+  idCarousel: PropTypes.any,
```
Comment (sebbalex) [inline src/components/DesktopSwiper.jsx:226]: same here

File: src/components/Tab.jsx [hunk 4]
```diff
@@ -355,4 +356,7 @@ Tab.propTypes = {
   tabContent: PropTypes.any,
   sideDesktopNavigation: PropTypes.bool,
   mobileNavigation: PropTypes.bool,
+  title: PropTypes.any,
+  subtitle: PropTypes.any,
+  sectionId: PropTypes.any,
```
Comment (sebbalex) [inline src/components/Tab.jsx:361]: same here

File: src/components/hero/Hero.jsx [hunk 5]
```diff
@@ -38,4 +38,6 @@ Hero.propTypes = {
   yPaddingXLScreen: PropTypes.bool,
   children: PropTypes.node.isRequired,
   className: PropTypes.any,
+  ariaLabelledBy: PropTypes.any,
+  Tag: PropTypes.any,
```
Comment (sebbalex) [inline src/components/hero/Hero.jsx:42]: same here

File: src/components/hero/HeroButton.jsx [hunk 6]
```diff
@@ -1,7 +1,17 @@
...
   </a>
 );
+
+HeroButton.propTypes = {
+  classButton: PropTypes.any,
+  href: PropTypes.any,
+  label: PropTypes.any,
+  ariaLabel: PropTypes.any,
+  rel: PropTypes.any,
+  target: PropTypes.any,
```
Comment (sebbalex) [inline src/components/hero/HeroButton.jsx:16]: same here

File: src/components/hero/HeroHowItWorks.jsx [hunk 7]
```diff
@@ -232,3 +233,10 @@ export const HeroHowItWorks = ({ title, body, image, list }) => {
     </Hero>
   );
 };
+
+HeroHowItWorks.propTypes = {
+  title: PropTypes.any,
+  body: PropTypes.any,
+  image: PropTypes.any,
+  list: PropTypes.any,
```
Comment (sebbalex) [inline src/components/hero/HeroHowItWorks.jsx:241]: same here

**Review by sebbalex — CHANGES_REQUESTED** [review]
PropTypes need to reflect correct type to be less error prone and have types safe checked where possible.

**Review by sebbalex — APPROVED** [review]

**Pull request merged**